packages/analytics-js-integrations/__tests__/integrations/MicrosoftClarity/browser.test.js

```
@@ -0,0 +1,566 @@
+      message: {
+        userId: 'test-user-123',
+        context: {},
+      },
+    };
+
+    clarityInstance.identify(rudderElement);
+    expect(window.clarity).toHaveBeenCalledWith('identify', 'test-user-123', undefined, undefined);
+  });
+
+  test('should handle when clarity identify returns an object with then property that is not a function', () => {
+    window.clarity = jest.fn().mockReturnValue({ then: 'not-a-function' });
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)